bee-runtime/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "bee-runtime"
+version = "0.0.0-alpha"
```

```suggestion
version = "0.1.0-alpha"
```
Let's release this today.